src/nylas-connection.ts

```diff
@@ -197,7 +195,17 @@ export default class NylasConnection {
           });
         } else {
           if (options.downloadRequest) {
-            return resolve(response.body);
+            response.buffer().then(buffer => {
+              // Return an object with the headers and the body as a buffer
+              const fileDetails: { [key: string]: any } = {};
+              response.headers.forEach((v, k) => {
```

Comment:
Not sure if this is necessary or not, but in the previous versions the SDK would return the headers in an object with the header, so I'm adding this to maintain backwards compatibility. Also `response.headers` is no longer a JS object but some sort of map so I can't just spread the headers into an object :/